bodhi/server/consumers/automatic_updates.py

```diff
@@ -108,7 +108,7 @@ def __call__(self, message: fedora_messaging.api.Message) -> None:
 
             bcls = ContentType.infer_content_class(Build, kbuildinfo)
             build = bcls.get(bnvr)
-            if build:
+            if build and build.update:
```

Comment:
This looks to be in the wrong place, we definitely want it to return on line 128 and not try to create the same build again if it exists.